### PR TITLE
Make README.md slightly more detailed and up-to-date

### PR DESCRIPTION
**Pull request by MGJerry**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -39,7 +48,7 @@ After many tedious searches trying to find a solution, I came across a save edit
 ## Compatibility
 
 Requirements:
-- No Man's Sky - FRONTIERS (PC/Xbox/PS4)
+- No Man's Sky - WAYPOINT(PC/Xbox/PS4)
```
Comment (Malix-Off) [inline README.md:51]:
```suggestion
- No Man's Sky - WAYPOINT (PC/Xbox/PS4)
```

**Comment by Malix-Off** [comment]
More Enhanced and Functional Compared to #613

**Review by Malix-Off — COMMENTED** [review]
**Misleading**: Application is not updated to the "WAYPOINT" version as of now

**Comment by MGJerry** [comment]
@Malix-Off check 1.11 update in [CHANGELOG.md](https://github.com/goatfungus/NMSSaveEditor/blob/master/CHANGELOG.md), it is compatible with up to game version 4.0.3 (WAYPOINT was version 4.0)

**However**, you do have a point in saying that it is, because technically the game's current version is 4.06
(don't ask me about NMS's version naming, I'm confused too)

**Pull request merged**